marlowe-playground-client/src/Web/HTML/WindowExtra.purs

```diff
@@ -0,0 +1,27 @@
+-- This module contains helpers that should eventually be added into purescript-web-html
+-- through a PR
+module Web.HTML.WindowExtra
+  ( close
+  , postMessage
+  ) where
+
+import Prelude
+import Effect (Effect)
+import Effect.Uncurried as FU
+import Foreign (Foreign)
+import Web.HTML (Window)
+
+foreign import _close :: FU.EffectFn1 Window Unit
+
+close :: Window -> Effect Unit
+close = FU.runEffectFn1 _close
+
+foreign import _postMessage :: FU.EffectFn2 Foreign Window Unit
+
+-- Post message dispatches a MessageEvent on a Window.
+-- Browser documentation: https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage
+-- TODO: targetOrigin and transfer are not being modeled, see if needed later
```

Comment:
perhaps we should follow the warning and force something

> Always provide a specific targetOrigin, not *, if you know where the other window's document should be located. Failing to provide a specific target discloses the data you send to any interested malicious site.

Reply:
done